tensorflow_addons/activations/rrelu_test.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from absl.testing import parameterized
+
+import math
+
+import numpy as np
+import tensorflow as tf
+from tensorflow_addons.activations import rrelu
+from tensorflow_addons.utils import test_utils
+import random
+
+SEED = 111111
+tf.random.set_seed(SEED)
+random.seed(SEED)
```

Comment:
Could we set seeds in testing methods?

Reply:
Should we use PhiloxRandom as random::UniformDistribution<random::PhiloxRandom, TYPE>> which has been used in tensorflow.